packages/editor-ui/src/components/WorkflowActivator.vue

```diff
@@ -100,9 +104,11 @@ export default mixins(
 						// workflow. If that would not happen then it could be quite confusing
 						// for people because it would activate a different version of the workflow
 						// than the one they can currently see.
-						const importConfirm = await this.confirmMessage(`When you activate the workflow all currently unsaved changes of the workflow will be saved.`, 'Activate and save?', 'warning', 'Yes, activate and save!');
-						if (importConfirm === false) {
-							return;
+						if (this.isDirty) {
```

Comment:
referring to previous comment
```suggestion
						if (this.dirtyState) {
```